torch/csrc/jit/codegen/cuda/compute_at_map.h

```diff
@@ -71,6 +71,8 @@ class TORCH_CUDA_CU_API ComputeAtMap {
   std::string toString();
 
  private:
+  bool has_lowered_kir_ = false;
```

Comment:
I assume the intention is to make `ComputeAtMap` available without lowering. Wouldn't it be simpler and cleaner if the mappings for KIR nodes are completely separated from the mappings for the Fusion IR? For example, how about letting `ComputeAtMap` only take care of the Fusion IR nodes, and create, say, class `kir::ComputeAtMap`, that is "lowered" from `ComputeAtMap`?

Reply:
@csarofeen  I updated the interface just a little bit in order to make things more explicit. `build` takes `Fusion*` and `GpuLower*` as its parameters. If `GpuLower*` is `nullptr`, it doesn't generate the KIR version of the mappings.

https://github.com/csarofeen/pytorch/pull/688/commits/94ee1ecf9664ed44f764074a141553f34bca80bf